src/control/settings/LatexSettings.h

```diff
@@ -20,8 +20,8 @@ class LatexSettings {
     bool autoCheckDependencies{true};
     fs::path globalTemplatePath{};
 #ifdef __APPLE__
-    std::string genCmd{"/Library/TeX/texbin/pdflatex -halt-on-error -interaction=nonstopmode '{}'"};
+    std::string genCmd{"/Library/TeX/texbin/pdflatex -halt-on-error --shell-escape -interaction=nonstopmode '{}'"};
 #else
-    std::string genCmd{"pdflatex -halt-on-error -interaction=nonstopmode '{}'"};
+    std::string genCmd{"pdflatex -halt-on-error --shell-escape -interaction=nonstopmode '{}'"};
```

Comment:
This additional flag should only be enabled for the Markdown template. This worsens the attack surface of the LaTeX template tremendously.

Reply:
Please note that the `-shell-escape` flag must only be enabled for pdflatex. With lualatex, it can be dropped completely.

Reply:
We'd have to look up if lualatex is available on all platforms we ship to. Also I don't really think it is an issue if we use `--shell-escape` for markdown, as it is quite secure with the Markdown in its separate file. My worries are solely with using this flag for LaTeX, as it allows literally any command being issued in the shell.

Reply:
The LuaTeX binary is [available since at least TeX Live 2016](http://www.luatex.org/). It predates the Markdown package, i.e. in a full TeX Live installation, LuaTeX should be available whenever the Markdown package is available.

Reply:
@xournalpp/core what is your opinion on switching to LuaTeX?

Reply:
I don't see any problem changing the default command as long as it produces non-zero error codes on build failures.

Reply:
luatex and lualatex are also included in TinyTeX (which we currently use for the snap package).